spec.html

```diff
@@ -43232,8 +43232,11 @@ <h1>Colophon</h1>
 <emu-annex id="sec-bibliography">
   <h1>Bibliography</h1>
   <ol>
-    <li>
-      IEEE Std 754-2008: <i>IEEE Standard for Floating-Point Arithmetic</i>. Institute of Electrical and Electronic Engineers, New York (2008)
+    <li id="sec-bibliography-ieee754">
```

Comment:
i wonder if instead of an li id, we could have a `<dfn>` around `IEEE Std 754-2019` and then it would get auto-linked?

Reply:
I didn't know about `<dfn>`. Fixed!